samples/subsys/ipc/ipc_service/static_vrings/src/main.c

```diff
@@ -231,14 +242,32 @@ static void ipc1_entry(void *dummy0, void *dummy1, void *dummy2)
 	k_sem_take(&ipc1_bound_sem, K_FOREVER);
 
 	while (message < 100) {
-		ret = ipc_service_send(&ipc1_ept, &message, sizeof(message));
+		uint32_t len = 0;
```

Comment:
If this line was changed to `len = sizeof(message);` you would capture the problem I've described.